governance-exercises/VoteWizard_poll_exercise_02/Derecognition of Clive AD Due to Compromised Operational Security - July 28, 2023.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Derecognition of Clive AD Due to Compromised Operational Security - JULY 28, 2023
+summary: Signal your support or opposition to derecognition of Clive AD due to compromised operational security. A community member, Diane, posted materials claiming that they could reveal the delegate's identity.
```

Comment:
Should remove second sentence here. The summary should still be primarily talking about the vote. Maybe the below:
```suggestion
summary: Signal your support or opposition to derecognition of Clive AD due to potentially compromised operational security, as reported publicly by Diane.
```

Really not convinced you need to mention Diane at all though.